src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -232,3 +232,29 @@ The strings can also be accessed directly, outside of the context of IoC referen
 ```javascript
 that.msgLookup.lookup(value); // where value is either the string name or the key in the stringArrayIndex to lookup.
 ```
+
+## Specifying a Localization
+
+The messageLoader takes `defaultLocale` and `locale` options for specifying which localized message bundle to fetch. The `locale` option specifies the localization desired. By default it is sourced from the prefsEditorLoader's settings object, `"{prefsEditorLoader}.settings.locale"`. The `defaultLocale` provides a fallback to use if the desired localization cannot be located. By default it is sourced from the prefsEditorLoader's `defaultLocale` option.
+
+### Fallback Rules
+
+If a requested localization cannot be found, the messageLoader will attempt to find another message bundle to satisfy the request. Language codes are expected in a form similar to BCP 47 tags but with a "\_" instead of a "-" separating the language code from the country code.
+
+Assuming the Canadian French message bundle is requested:
+1. look for a message bundle corresponding to the language code specified by the `locale` option (e.g. "fr_CA")
+2. look for a message bundle with the same language as the language code specified by the `locale` option (e.g. "fr")
+3. look for a message bundle corresponding to the language code specified by the `defaultLocale` option (e.g. "en_US")
+4. look for a message bundle with the same language as the language code specified by the `defaultLocale` option (e.g. "en")
+5. look for a message bundle with the exact URL as specified through the auxiliary schema or directly to the messageLoader (e.g. "message/prefsEditor.json")
```

Comment:
Remove "or directly to the messageLoader" if we decide not to mention the grade version of prefs framework.